graphql-server/go-server/main.go

```diff
@@ -70,7 +72,15 @@ func main() {
 	})
 
 	if *enablePlayground {
-		http.Handle("/", playground.Handler("Arcadia-Graphql-Server", "/bff"))
+		endpoint := "/bff"
+		if *ingressPrefix != "" {
+			prefix := strings.TrimPrefix(*ingressPrefix, "/")
+			prefix = strings.TrimSuffix(prefix, "/")
+			if prefix != "" {
+				endpoint = fmt.Sprintf("/%s%s", prefix, endpoint)
```

Comment:
endpoint := fmt.Sprintf("%s/bff", strings.TrimSuffix(strings.TrimPrefix(*ingressPrefix, "/"), "/")

@0xff-dev will this work?

Reply:
or just pass `--playgroundEndpoint` ???

Reply:
https://github.com/99designs/gqlgen/blob/132ec1ce579e9ce3dc772afebf9703c1403d588e/graphql/playground/playground.go#L53

https://github.com/99designs/gqlgen/blob/132ec1ce579e9ce3dc772afebf9703c1403d588e/graphql/playground/playground.go#L118

According to these two functions, endpoint must start with a /. If playgroundEndpointPrefix is written with only a /, the path to endpoint will be incorrect as written above.

```go
prefix := strings.TrimSuffix(strings.TrimPrefix(*ingressPrefix, "/"), "/")
if prefix != "" {
    endpoint = fmt.Sprintf("/%s%s", prefix, endpoint)
}
```

Reply:
we need a common name for this endpoint definition. How about use `--playgroundEndpoint` with `/bff` by default